1-resilient-messaging/src/Application/OrderService.php

```diff
@@ -7,16 +7,27 @@
 use App\Domain\Order;
 use App\Domain\OrderRepository;
 use App\Domain\ShippingService;
+use App\Event\OrderWasPlaced;
+use Ecotone\Messaging\Attribute\Asynchronous;
 use Ecotone\Modelling\Attribute\CommandHandler;
-
+use Ecotone\Modelling\Attribute\EventHandler;
+use Ecotone\Modelling\EventBus;
+use App\Application\PlaceOrder;
 final class OrderService
 {
-    #[CommandHandler]
-    public function placeOrder(PlaceOrder $placeOrder, OrderRepository $orderRepository, ShippingService $shippingService): void
+    #[Asynchronous("orders")]
```

Comment:
W `run_example.php` jest konkretna ilość wywołań Consumer'a, który przetwarza wiadomości z kolejki. 
Uwzględniłem tylko, że ktoś oznaczy Event Handler'a jako `async`, a Ty dołożyłeś Command Handlera, więc jedna wiadomość więcej niż zakładałem :P 

Możesz to dropnąć i zostawić Command Handlera synchronicznego (poprawiłem też README.md).